YOLO.xcodeproj/project.pbxproj

```diff
@@ -210,13 +195,8 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				6381D21B2B7817C200ABA4E8 /* yolov8m.mlpackage in Sources */,
```

Comment:
Again, we don't want to eliminate these references!

Reply:
I changed those lines back and restored the model references.